nav2_behavior_tree/plugins/action/clear_costmap_service.cpp

```diff
@@ -29,6 +30,10 @@ ClearEntireCostmapService::ClearEntireCostmapService(
 
 void ClearEntireCostmapService::on_tick()
 {
+  std::vector<std::string> plugins;
```

Comment:
The `plugins` local variable now unused and can be removed. Also in all 3 other instances in this file